scss/_tags.scss

```diff
@@ -0,0 +1,159 @@
+// stylelint-disable function-disallowed-list
+
+.tag {
+  // scss-docs-start tag-css-vars
+  --#{$prefix}tag-gap: #{$tag-gap};
+  --#{$prefix}tag-padding-x: #{$tag-padding-x};
+  --#{$prefix}tag-padding-y: #{$tag-padding-y};
+  --#{$prefix}tag-margin: 0;
```

Comment:
I don't see why we'd need to have a margin customization here. Margins are generally defined thanks to a context so one could simply use our utilities no?

Reply:
I've added it to override default paragraph margin but could be set to 0 and customized with utilities indeed